backtester/eventhandlers/exchange/exchange.go

```diff
@@ -119,9 +119,9 @@ func (e *Exchange) ExecuteOrder(o order.Event, dh data.Handler, om *engine.Order
 	}
 
 	if cs.CanUseExchangeLimits || cs.UseRealOrders {
-		// Conforms the amount to the exchange order defined step amount
+		// Validate the amount to the exchange order defined step amount
 		// reducing it when needed
```

Comment:
🚧 I don't think this comment reflects what is happening
But I also don't think it's adding any information over the well defined method name
Suggest removing.
```suggestion
```

Oh, I see what happened. You were going to use Validate, but then switched it out.